ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -524,10 +551,22 @@ else
 				then
 				else
 					echo "FMU mode_${FMU_MODE} start failed" >> $LOG_FILE
-					tune_control play -m ${TUNE_ERR}
+					# Error tune.
+					tune_control play -t 2
 				fi
 			fi
 		fi
+
+		if [ $OUTPUT_MODE == fmu ]
```

Comment:
Did you see my comment here? Unfortunately it got hidden on github due to additional changes.

> The above fmu start is also inside the else of if [ $IO_PRESENT == yes ], which means there is a case where the fmu is not started (if $IO_PRESENT == yes && $OUTPUT_MODE != fmu) and thus we cannot simply drop/merge it.
> Did I understand you correctly?
> Also I think the startup ordering between io and fmu is important, we'd have to double-check that if we change it.

Reply:
I missed attending to that, Done. Thanks!